scripts/build/builders/efr32.py

```diff
@@ -259,27 +261,64 @@ def __init__(self,
     def GnBuildArgs(self):
         return self.extra_gn_options
 
+    def _bundle(self):
+        # Only unit-test needs to generate the flashbundle here.  All other examples will generate a flashbundle via the silabs_executable template.
+        if self.app == Efr32App.UNIT_TEST:
+            flash_bundle_path = os.path.join(self.output_dir, self.app.FlashBundleName())
+            logging.info(f'Generating flashbundle {flash_bundle_path}')
+
+            patterns = [
+                os.path.join(self.output_dir, "tests", "*.flash.py"),
+                os.path.join(self.output_dir, "tests", "*.s37"),
+                os.path.join(self.output_dir, "tests", "silabs_firmware_utils.py"),
+                os.path.join(self.output_dir, "tests", "firmware_utils.py"),
+            ]
+
+            # Generate the list of files by globbing each pattern.
+            files = []
+            for pattern in patterns:
+                files += list(map(lambda x: os.path.basename(x), glob.glob(pattern)))
```

Comment:
nit: list + map is often easier to read in a `[foo(x) for x in y]` (instead of `list(map(lambda x: foo(x), y))`).

So here it may be:

```py
files.extend([os.path.basename(x) for x in glob.glob(pattern)])
```

and below it may be:

```py
for name in [os.path.basename(x) for x in glob.glob(pattern)]:
```